FRBDK/Glue/OfficialPlugins/AnimationChainPlugin/ViewModels/AnimationFrameViewModel.cs

```diff
@@ -81,7 +85,19 @@ public void SetFrom(AnimationChainViewModel parent, AnimationFrameSave animation
             ResolutionWidth = resolutionWidth;
             ResolutionHeight = resolutionHeight;
 
+            foreach(var rect in animationFrame.ShapeCollectionSave?.AxisAlignedRectangleSaves)
```

Comment:
I don't think you can do null coalesce  `?.` in a foreach. I believe this will blow up.